GitUI/CommandsDialogs/RevisionDiff.cs

```diff
@@ -164,21 +155,56 @@ protected override void OnRuntimeLoad(EventArgs e)
 
         private string DescribeRevision(string sha1)
         {
+            return DescribeRevision(sha1, 0);
+        }
+
+        private string DescribeRevision(string sha1, int maxLength)
```

Comment:
we can combine these two methods into one - `private string DescribeRevision(string sha1, int maxLength = 0)`

Reply:
changing